user_guide_src/source/installation/upgrade_4xx.rst

```diff
@@ -1,114 +1,102 @@
-have to move the socket path from ``$config['socket']`` to
-``$config['host']`` instead.
-
-The ``$config['socket_type']`` option is also removed, although that won't
-affect your application - it will be ignored and the connection type will
-be determined by the format used for ``$config['host']`` instead.
-
-Memcache(d)
------------
-
-The 'memcached' will now ignore configurations that don't specify a ``host``
-value (previously, it just set the host to the default '127.0.0.1').
-
-Therefore, if you've added a configuration that only sets e.g. a ``port``,
-you will now have to explicitly set the ``host`` to '127.0.0.1' as well.
-
-Step 5: Check usage of doctype() HTML helper
-============================================
-
-The :doc:`HTML Helper <../helpers/html_helper>` function
-:php:func:`doctype()` used to default to 'xhtml1-strict' (XHTML 1.0 Strict)
-when no document type was specified. That default value is now changed to
-'html5', which obviously stands for the modern HTML 5 standard.
-
-Nothing should be really broken by this change, but if your application
-relies on the default value, you should double-check it and either
-explicitly set the desired format, or adapt your front-end to use proper
-HTML 5 formatting.
\ No newline at end of file
+CodeIgniter 4 is a rewrite of the framework, and is not backwards compatible.
+It is more appropriate to think of converting your app, rather than upgrading it.
+Once you have done that, upgrading from one version of CodeIgniter 4 to the next
+will be straightforward.
+
+The "lean, mean and simple" philosophy has been retained, but the 
+implementation has a lot of differences, compared to CodeIgniter 3.
+
+There is no 12-step checklist for upgrading. Instead, start with a copy
+of CodeIgniter 4 in a new project folder, `however you wish to install and 
+use it </installation>`_, 
+and then convert and integrate your app components.
+We'll try to point out the most important considerations here.
+
+Not all of the CI3 libraries have been ported or rewritten for CI4!
+See the threads in the `CodeIgniter 4 Roadmap <https://forum.codeigniter.com/forum-33.html>`_
+subforum for an uptodate list!
+
+**Downloads**
+- CI4 is still available as a ready-to-run zip or tarball, which
+includes the user guide (though in the `docs` subfolder
+- It can also be installed using Composer
+
+**Namespaces**
+- CI4 is built for PHP7.1+, and everything in the framework is namespaced, even the helpers
+
+**Application Structure**
+- The framework still has `application` and `system` folders, with the same 
+interpretation as before
+- The framework now provides for a `public` folder, intended as the document
+root for your app
+- There is also a `writable` folder, to hold cache data, logs, and session data
+- The `application` folder looks very similar to that for CI3, with some
+name changes (eg Filters instead of hooks) and capitalization, and some subfolders
```

Comment:
Events replaced Hooks, not filters.

Reply:
I will stick to mentioning changed names :-/